app/app.iml

```diff
@@ -17,17 +17,15 @@
       <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
       <sourceFolder url="file://$MODULE_DIR$/gen" isTestSource="false" generated="true" />
     </content>
-    <orderEntry type="jdk" jdkName="Android API 21 Platform" jdkType="Android SDK" />
+    <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
-    <orderEntry type="library" name="libs" level="project" />
     <orderEntry type="module" module-name="achartengine" />
     <orderEntry type="module" module-name="android-mapviewballoons" />
+    <orderEntry type="module" module-name="android-GridViewWithHeaderAndFooter" />
     <orderEntry type="module" module-name="backend" />
     <orderEntry type="module" module-name="cases" />
     <orderEntry type="module" module-name="core" />
-    <orderEntry type="module" module-name="appcompat" />
-    <orderEntry type="library" name="android-support-v7-appcompat" level="project" />
-    <orderEntry type="library" name="android-support-v42" level="project" />
+    <orderEntry type="library" name="odk-libs" level="project" />
```

Comment:
Requires proactive renaming of library that odk uses to build in the project settings.